app/Resources/views/subscription/update_overview.html.twig

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html.twig" %}
+
+{% block body %}
+    <h2>{{ 'overview.update.subtitle'|trans }}</h2>
+
+    <p>
+        {{ 'overview.update.intro'|trans|raw }}
```

Comment:
Jah, standaard CSRF form protection van SF. Ook is er een 'purified_textarea' type gebruikt welke HTMLPurifier lib gebruikt.